sidebars.js

```diff
@@ -110,6 +110,7 @@ module.exports = {
       "api/contacts-group/delete"
     ],
     "Import/Export": [
+      "api/import-export/overview",
       "api/import-export/create", 
       "api/import-export/read", 
       "api/import-export/update", 
```

Comment:
This file has been deleted as there isnt an update specific endpoint to send the request to within the API. Removing this file also requires the link reference to be removed from the sidebar other the app fails to start up properly

Reply:
We can also remove the delete entry as there isnt any metadata that can be deleted with this feature. Deletions will need to happen on an individual basis per category